cpu/native/startup.c

```diff
@@ -325,10 +361,23 @@ __attribute__((constructor)) static void startup(int argc, char **argv)
 
     native_cpu_init();
     native_interrupt_init();
+#if defined(MODULE_NETDEV2_TAP) || defined(MODULE_NETDEV2_RAW802154)
+    /* configure signal handler for fds */
+    native_async_read_setup();
+#endif
 #ifdef MODULE_NETDEV2_TAP
-    netdev2_tap_params_t p;
-    p.tap_name = &(argv[1]);
-    netdev2_tap_setup(&netdev2_tap, &p);
+    if (tap_dev) {
+        netdev2_tap_params_t p;
+        p.tap_name = &(argv[1]);
+        netdev2_tap_setup(&netdev2_tap, &p);
+    }
```

Comment:
There is a semantic change here in that the interface is optional now. It was a positional parameter before because a network application can't run without it.
If there is good reason (other than personal preference) to discard the positional parameter please explain.